src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/payment_method/update.yaml

```diff
@@ -0,0 +1,42 @@
+twig_hooks:
+    hooks:
+        'sylius_admin.payment_method.update.content':
+            form:
+                component: 'sylius_admin:payment_method:form'
+                data:
+                    payment_method: '@=resource'
+                    form: '@=form'
+                    parentMainHook: '@=parent_main_hook'
+                    parentFallbackHook: '@=parent_fallback_hook'
+                configuration:
+                    method: 'PUT'
+
+        'sylius_admin.payment_method.update.content.form':
+            sections:
+                template: '@SyliusAdmin/payment_method/form/sections.html.twig'
+
+        'sylius_admin.payment_method.update.content.form.sections':
+            general:
+                template: '@SyliusAdmin/payment_method/form/sections/general.html.twig'
+            gateway_configuration:
+                template: '@SyliusAdmin/payment_method/form/sections/gateway_configuration.html.twig'
+            translations:
+                template: '@SyliusAdmin/payment_method/form/sections/translations.html.twig'
+
+        'sylius_admin.payment_method.update.content.form.sections.general':
+            code:
+                template: '@SyliusAdmin/payment_method/form/sections/general/code.html.twig'
+            position:
+                template: '@SyliusAdmin/payment_method/form/sections/general/position.html.twig'
+            enabled:
+                template: '@SyliusAdmin/payment_method/form/sections/general/enabled.html.twig'
+            channels:
+                template: '@SyliusAdmin/payment_method/form/sections/general/channels.html.twig'
+
+        'sylius_admin.payment_method.update.content.form.sections.gateway_configuration':
+            type:
+                template: '@SyliusAdmin/payment_method/form/sections/gateway_configuration/type.html.twig'
+
+        'sylius_admin.payment_method.update.content.form.sections.translations':
```

Comment:
There are missing hooks for `paypal` and `stripe` configuration.

Reply:
Fixed 🔧